### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DanRoman-code**

### CODE REVIEW

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_of_correctly_type_and_result():
```
Comment (IvanRamyk) [inline app/test_main.py:4]: Please, add tests for the boundary values

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

**Review by IvanRamyk — APPROVED** [review]
Good job!
